lambdas/src/apis/status/PeerHealthStatus.ts

```diff
@@ -12,13 +11,12 @@ export default class PeerHealthStatus {
   private contentServerStatus: TimeRefreshedDataHolder<HealthStatus>
   private commsServerStatus: TimeRefreshedDataHolder<HealthStatus>
   private lambdaServerStatus: TimeRefreshedDataHolder<HealthStatus>
-  private commsServerUrl: string
 
   constructor(
     contentClient: SmartContentClient,
     maxSynchronizationTime: string,
     maxDeploymentObtentionTime: string,
-    private readonly externalCommsServerAddress: string
+    private readonly commsServerAddress: string
```

Comment:
Where is this constructor being called? Do we need to update to receive this new parameter?

Reply:
This was used  [here](https://github.com/decentraland/catalyst/blob/f2e4aa49229bc9bb6e082a1113f5609b2ccdf51c/lambdas/src/apis/status/routes.ts#L11) and since the variable is still named the same, it works